server/game/cards/08.1-TAK/NaggasRibs.js

```diff
@@ -0,0 +1,26 @@
+const DrawCard = require('../../drawcard.js');
+
+class NaggasRibs extends DrawCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            effect: ability.effects.modifyDominanceStrength(2 * this.controller.deadPile.size())
```

Comment:
Might be a tad paranoid, but this will break if we'll ever have non-character cards in the dead pile. If we want to be defensive, this should count only character cards. Not sure if it's worth it.